### PR TITLE
[UK] Add Project Loved

### PR DESCRIPTION
**Pull request by GeraldAzgore**
i make a ukrainian translate for "Project Loved"

### CODE REVIEW

File: wiki/Project_Loved/ru.md [hunk 1]
```diff
@@ -1,48 +1,48 @@
 # Project Loved
 
-**Project Loved** (*проект «Любимые карты»*, также *«Лавд»*) — это платформа для голосования, через которую карты попадают в категорию [Loved](/wiki/Beatmaps#loved). Проектом управляет [одноимённая команда](/wiki/People/The_Team/Project_Loved_Team).
+**Project Loved** (*проект «Улюблені Мапи»*, також *«Лавед»*) — це платформа для голосування, через яку мапи потрапляють у категорію [Loved](/wiki/Beatmaps#loved). Проектом управляє [одноименна команда](/wiki/People/The_Team/Project_Loved_Team).
 
-## Продвижение карты в Loved
+## Просування карти у Loved
 
-Карты получают статус Loved путём номинирования и голосования.
+Мапи отримують статус Loved шляхом номінування та голосування.
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:7]:
```suggestion
Карти отримують статус Loved шляхом номінування та голосування.
```
Думаю, лучше будет звучать, да и более красиво

File: wiki/Project_Loved/ru.md [hunk 2]
```diff
@@ -1,48 +1,48 @@
...
 
-Члены команды Project Loved, называемые **капитанами**, регулярно номинируют карты для различных игровых режимов. Почти всегда эти карты предлагаются сообществом osu!. Актуальные формы заявок и списки карт, уже находящихся в Loved, можно найти по следующим ссылкам:
+Члени команди Project Loved, названні **капітанами**, регулярно номінують мапи для різних ігрових режимів. Майже завжди ці карти пропонуються спільнотою osu!. Актуальні форми заявок та списки мап, вже знаходящихся в Loved, можна знайти по слідуючим посиланням:
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:9]:
```suggestion
Члени команди Project Loved, названні **капітанами**, регулярно номінують мапи для різних ігрових режимів. Майже завжди ці карти пропонуються спільнотою osu!. Актуальні форми заявок та списки біткарт, котрі вже знаходяться в Loved, можна знайти по наступним посиланням:
```
?? Очевидные ошибки, как-будто машинный перевод

File: wiki/Project_Loved/ru.md [hunk 3]
```diff
@@ -1,48 +1,48 @@
...
 
 - [Форма заявки для osu! и osu!catch](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform);
 - [Форма заявки для osu!taiko](https://docs.google.com/forms/d/e/1FAIpQLSclPWyjFByhHP45AtKD49y0RSl1TK5UOzD4dVdvjfJJQ1aCXQ/viewform);
 - [Форма заявки для osu!mania](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform);
-- [Список любимых карт в osu! и osu!catch](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit);
-- [Список любимых карт в osu!taiko](https://docs.google.com/spreadsheets/d/1Nk96z_xat_7ypsDF1sCTDO4i_CnHarcrCbGoTmgwHHE/edit);
-- [Список любимых карт в osu!mania](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit).
+- [Список улюблених мап у osu! и osu!catch](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit);
+- [Список улюблених мап у osu!taiko](https://docs.google.com/spreadsheets/d/1Nk96z_xat_7ypsDF1sCTDO4i_CnHarcrCbGoTmgwHHE/edit);
+- [Список улюблених мап у osu!mania](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit).
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:16]:
```suggestion
- [Список улюблених карт у osu! и osu!catch](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit);
- [Список улюблених карт у osu!taiko](https://docs.google.com/spreadsheets/d/1Nk96z_xat_7ypsDF1sCTDO4i_CnHarcrCbGoTmgwHHE/edit);
- [Список улюблених карт у osu!mania](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit).
```
Личные придирки, но лучше звучит

File: wiki/Project_Loved/ru.md [hunk 4]
```diff
@@ -1,48 +1,48 @@
...
 
-*Примечание: если у вас ограничен доступ к Google Формам, вы можете связаться с любым членом [команды Project Loved](/wiki/People/The_Team/Project_Loved_Team#члены-команды), чтобы карту представили от вашего имени.*
+*Примітка: якщо у вас обмежений доступ до Google Форм, ви можете зв'язатися з будь-яким членом [команди Project Loved](/wiki/People/The_Team/Project_Loved_Team#члены-команды), щоб мапу представили від вашого ім'я.*
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:18]:
```suggestion
*Примітка: якщо у вас обмежений доступ до Google Форм, ви можете зв'язатися з будь-яким членом [команди Project Loved](/wiki/People/The_Team/Project_Loved_Team#члены-команды), щоб мапу представили від вашого імені.*
```
Правила украинского языка

File: wiki/Project_Loved/ru.md [hunk 5]
```diff
@@ -1,48 +1,48 @@
...
 
-После сбора заявок номинированные карты будут выложены на [форумe Project Loved](https://osu.ppy.sh/community/forums/120), где остальные игроки смогут проголосовать, хотят ли они видеть карту в категории Loved. Карты, достигшие определенного процента голосов «за» к концу голосования, будут перемещены в категорию Loved.
+Після сбору заявок номінованні мапи будуть викладені на [форумі Project Loved](https://osu.ppy.sh/community/forums/120), де інші гравці зможуть проголосувати, чи хотять вони бачити цю мапу у категорії Loved. Мапи, досягнувщі відповідного проценту голосів «за» к кінцю голосування, будуть переміщенні до категорії Loved.
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:20]:
```suggestion
Після сбору заявок номінованні мапи будуть викладені на [форум Project Loved](https://osu.ppy.sh/community/forums/120), де інші гравці зможуть проголосувати за те, чи хочуть вони бачити цю мапу у категорії Loved. Карти, котрі досягли відповідного проценту голосів «за» до кінця голосування, будуть переміщенні до категорії Loved.
```
Исправлены некоторые ошибки

File: wiki/Project_Loved/ru.md [hunk 6]
```diff
@@ -1,48 +1,48 @@
...
 
-### Требования
+### Вимоги
 
-Карта, которую номинируют на добавление в Loved, должна соответствовать перечисленным требованиям:
+Мапа, яку номинують на додавання у Loved, повинна відповідати переліченим вимогам:
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:24]:
```suggestion
Карта, яку номінують на додавання у Loved, повинна відповідати переліченим критеріям:
```
"Критеріям" будет лучше звучать, думаю, да и более красноречиво

File: wiki/Project_Loved/ru.md [hunk 7]
```diff
@@ -1,48 +1,48 @@
...
 
-- Она не находится в категориях [Qualified](/wiki/Beatmaps#qualified), [Ranked](/wiki/Beatmaps#ranked) или [Approved](/wiki/Beatmaps#approved).
-- Создатель карты согласен с тем, чтобы его карта попала в категорию Loved.
-- Создатель карты не забанен или не имеет рестрикт.
-- Как минимум одна сложность может быть пройдена.
-- Карту должны добавить в избранное не менее 30 человек (относится только к osu!standard).
+- Вона не знаходится у категоріях [Qualified](/wiki/Beatmaps#qualified), [Ranked](/wiki/Beatmaps#ranked) або [Approved](/wiki/Beatmaps#approved).
+- Творець мапи згоден з тим, щоб його мапа потрапила у категорію Loved.
+- Творець мапи не заблокований або не має рестрикт.
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:28]:
```suggestion
- Творець мапи не заблокований, або не має ніяких [обмежень облікового запису](/wiki/uk/Help_Centre#restriction).
```
"Рестрикт" - нет такого слова, добавлена ссылка на вики, добавлена запятая

File: wiki/Project_Loved/ru.md [hunk 8]
```diff
@@ -1,48 +1,48 @@
...
+- Як минімум одна сложність може бути пройдена.
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:29]:
```suggestion
- Як мінімум одна складність може бути пройдена.
```
??

File: wiki/Project_Loved/ru.md [hunk 9]
```diff
@@ -1,48 +1,48 @@
...
+- Мапу повинні додати у обране не менш ніж 30 людей (відноситься тільки до osu!standard).
 
-Несмотря на то, что почти все карты из категории [Pending](/wiki/Beatmaps#pending) соответствуют этим критериям, из-за более строгих требований к номинации далеко не все из них попадают в категорию Loved.
+Не дивлячись на те, що майже усі мапи з категорії [Pending](/wiki/Beatmaps#pending) відповідають цим крітеріям, из-за більш строгих вимог до номинації далеко не всі з них потрапляють у категорію Loved.
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:32]:
```suggestion
Не дивлячись на те, що майже усі мапи з категорії [Pending](/wiki/Beatmaps#pending) відповідають цим крітеріям, із-за більш строгих вимог до номінації далеко не всі з них потрапляють у категорію Loved.
```

File: wiki/Project_Loved/ru.md [hunk 10]
```diff
@@ -1,48 +1,48 @@
...
 
-Карты, входящие в категорию Loved, также должны соответствовать нескольким наборам правил:
+Мапи,які входять у категорію Loved, також повинні відповідати кільком наборам правил:
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:34]:
```suggestion
Карти, які входять у категорію Loved, також повинні відповідати кільком критеріям:
```

File: wiki/Project_Loved/ru.md [hunk 11]
```diff
@@ -1,48 +1,48 @@
...
 
-- [Раздел «Метаданные» в критериях ранкинга](/wiki/Ranking_Criteria#метаданные),
-- [Правила содержимого песни](/wiki/Rules/Song_Content_Rules),
-- [Критерии визуального содержания](/wiki/Rules/Visual_Content_Considerations).
+- [Розділ «Метаданні» у крітеріях ранкингу](/wiki/Ranking_Criteria#метаданные),
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:36]:
```suggestion
- [Розділ «Метаданні» у крітеріях ранкінгу](/wiki/uk/Ranking_Criteria#metadata),
```

File: wiki/Project_Loved/ru.md [hunk 12]
```diff
@@ -1,48 +1,48 @@
...
+- [Правила вмісту пісні](/wiki/Rules/Song_Content_Rules),
+- [Крітерії визуального вмісту](/wiki/Rules/Visual_Content_Considerations).
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:38]:
```suggestion
- [Критерії візуального вмісту](/wiki/Rules/Visual_Content_Considerations).
```

File: wiki/Project_Loved/ru.md [hunk 13]
```diff
@@ -1,48 +1,48 @@
...
 
-Любые проблемы, связанные с этими правилами, обычно решаются во время голосования.
+Будь-які проблеми, зв'язанні з цими правилами, зазвичай вирышуються під час голосування.
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:40]:
```suggestion
Будь-які проблеми, зв'язанні з цими правилами, зазвичай вирішуються під час голосування.
```

File: wiki/Project_Loved/ru.md [hunk 14]
```diff
@@ -1,48 +1,48 @@
...
 
-### Приоритет номинации
+### Приорітет номинації
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:42]:
```suggestion
### Приорітет номінації
```

File: wiki/Project_Loved/ru.md [hunk 15]
```diff
@@ -1,48 +1,48 @@
...
 
-Строгих правил и критериев нет, но обычно номинированные карты хорошо известны и ценятся сообществом, что следует из названия категории. Кроме того, шанс попадания в Loved повышается, если мапсет давно существует или содержит оригинально замапанные сложности высокого качества.
+Строгих правил та крітеріїв немає, але зазвичай номінованні мапи добре відомі та цінуються спільнотою, що слідує з назви категорії. Крім того, шанс потрапляння у Loved повищується, якщо мапсет існує давно або вміщує оригінально замапанні сложністі високої якості.
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:44]:
```suggestion
Строгих правил та крітеріїв немає, але зазвичай номіновані карти добре відомі та цінуються спільнотою, що слідує з назви категорії. Крім того, шанс потрапляння у Loved підвищується, якщо мапсет існує давно або вміщує оригінально замапанні складності високої якості.
```

File: wiki/Project_Loved/ru.md [hunk 16]
```diff
@@ -1,48 +1,48 @@
...
 
-## Дополнительная информация
+## Додаткова Інформація
```
Comment (atimonder1) [inline wiki/Project_Loved/ru.md:46]:
```suggestion
## Додаткова інформація
```

**Review by atimonder1 — CHANGES_REQUESTED** [review]
Много ошибок, серьёзно :/

**Comment by GeraldAzgore** [comment]
@TPGPL why "state:Blocked"?

**Comment by TPGPL** [comment]
because you made changes to the Russian translation (`ru.md`) instead of making a new `ua.md` file; will remove the label after you apply the reviews above and I fix that problem on my end

**Comment by atimonder1** [comment]
> because you made changes to the Russian translation (`ru.md`) instead of making a new `ua.md` file; will remove the label after you apply the reviews above and I fix the problem on my end

*`uk.md`, since the Ukrainian locale is "uk", but not "ua"

**Comment by TPGPL** [comment]
> > because you made changes to the Russian translation (`ru.md`) instead of making a new `ua.md` file; will remove the label after you apply the reviews above and I fix the problem on my end
> 
> *`uk.md`, since the Ukrainian locale is "uk", but not "ua"

oh right i forgot about that, thanks for correcting me out

**Comment by TicClick** [comment]
the reviewer says the translation is okay now

**Review by TicClick — APPROVED** [review]

**Comment by atimonder1** [comment]
> the reviewer says the translation is okay now

I forgot - in the "load more" tab he did not accept my reviews, can you accept them, please?

**Comment by TicClick** [comment]
> I forgot - in the "load more" tab he did not accept my reviews, can you accept them, please?

this pull request is now merged and inactive, so someone needs to make the edits and open a new one. that could be you, @GeraldAzgore, or me myself for not checking that the review is fully applied

**Comment by TicClick** [comment]
> I forgot - in the "load more" tab he did not accept my reviews, can you accept them, please?

fixed in #4638

**Pull request merged**